Modules/Core/Common/include/itkVector.h

```diff
@@ -137,6 +137,11 @@ class ITK_TEMPLATE_EXPORT Vector : public FixedArray<T, NVectorDimension>
     : BaseArray(r)
   {}
 
+  /** Explicit constructor for std::array. */
+  explicit Vector(const std::array<ValueType, NVectorDimension> & stdArray)
```

Comment:
Thank you for making and submitting this change.  Would you say more about why you specified the constructor as `explicit`?  In particular, if there is a use case where implicit use of this constructor is prone to errors, what is it? -- and perhaps add the explanation to `itkVector.h`.

Reply:
>  Would you say more about why you specified the constructor as explicit? 

@Leengit Thanks for asking  😃 Suppose there is a function that has an `itk::Point` and an `itk::Vector` as argument.  For example:

    // Example
    Point<double> AddVector(Point<double>, Vector<double>);

If both `itk::Point` and an `itk::Vector`  would have an implicit constructor, converting from `std::array`, a user who would pass two arrays as arguments might accidentally pass the arrays in the wrong order. Because of the `explicit` keyword, the user must be explicit about which `std::array` has the point coordinates, and which array has the vector elements:  

    // User code:
    const auto point = AddVector(Point<double>(stdArray1), Vector<double>(stdArray2));

Conceptually, `itk::Vector` and `itk::Point` are more specific than the general purpose `std::array`. I think it's good practice to make conversion from a more general data type to a more specific type explicit. Do you agree?